aten/src/ATen/PTThreadPool.h

```diff
@@ -4,6 +4,16 @@
 
 namespace at {
 
+// Checks external number of threads setting and returns
+// default value (hardware_concurrency()) if it's negative
+inline size_t check_and_get_pool_size(int nthreads) {
```

Comment:
does it need to be an interface function? feels sketchy

Reply:
I need to put the 'default thread pool size' somewhere, can move it